files/zh-cn/learn/javascript/objects/classes_in_javascript/index.html

```diff
@@ -339,10 +339,10 @@ <h3 id="Inheritance_with_class_syntax">Inheritance with class syntax</h3>
 snape.subject; // Dark arts
 </pre>
 
-<p>Like we did with Teachers, we could create other subclasses of <code>Person</code> to make them more specialized without modifying the base class.</p>
+<p>就像我们对教师所做的那样，我们可以创建 <code>Person</code> 的其他子类，以使它们更加专业，而无需修改基类。</p>
```

Comment:
上面的 `Teacher` 是一个类，这里不应该翻译。

```suggestion
<p>就像我们对 Teacher 所做的那样，我们可以创建 <code>Person</code> 的其他子类，以使它们更加专业，而无需修改基类。</p>
```